enterprise/server/cmd/ci_runner/run_local.sh

```diff
@@ -50,20 +50,14 @@ TEMPDIR=$(mktemp --dry-run | xargs dirname)
 : "${PERSISTENT:=true}"
 : "${BUILDBUDDY_API_KEY:=}"
 
-bazel build //enterprise/server/cmd/ci_runner:buildbuddy_ci_runner
-RUNNER_PATH="$PWD/bazel-bin/enterprise/server/cmd/ci_runner/buildbuddy_ci_runner"
-RUNNER_DATA_DIR="$TEMPDIR/buildbuddy_ci_runner_data"
-mkdir -p "$RUNNER_DATA_DIR"
-rm -f "$RUNNER_DATA_DIR/ci_runner"
-cp "$RUNNER_PATH" "$RUNNER_DATA_DIR/ci_runner"
-echo "Copied ci_runner to $RUNNER_DATA_DIR/ci_runner"
-
 mkdir -p "$CI_RUNNER_BAZEL_CACHE_DIR"
 
+# Build the ci_runner_image locally an load it into Docker daemon
+bazel run enterprise/server/cmd/ci_runner:ci_runner_image
+
```

Comment:
this script is a bit old so Im not too sure on how to update it accordingly 🤔

Reply:
This script isn't very useful anymore and isn't actively maintained so feel free to delete it. These days it's more convenient to use `bb remote` or `ExecuteWorkflow` (either via API or UI) to test the CI runner locally.